.github/workflows/test-reporter.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   report:
     runs-on: ubuntu-latest
     steps:
-      - uses: elastic/apm-pipeline-library/.github/actions/test-report@current
+      - uses: elastic/apm-pipeline-library/.github/actions/test-report@feature/update-test-reporter-action
```

Comment:
To be changed with this branch so we can verify it. works as expected then we can revert it. This workflow needs to exist in the default branch